src/app/GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -439,7 +439,10 @@ protected override void Dispose(bool disposing)
         {
             if (disposing)
             {
-                _repositoryHistoryUIService.GitModuleChanged -= SetGitModule;
+                if (_repositoryHistoryUIService is not null)
+                {
```

Comment:
No, it's nullable as it's initialised via `Initialize` method. Because this is a UI element which is instantiated via the designer, we can't use non-default .ctors.
The GitUI project isn't NRTA-enabled, so the nullable annotations aren't enforced... Thank you for spotting, I'll have the annotation fixed.

Reply:
This one was failing with NRE under the translationapp... I didn't look too deeply into this.